File: src/Library/ObjectModel/SoundbridgeEventArgs.vb
Public Class SoundbridgeEventArgs
    Inherits System.EventArgs

    Private _soundbridge As Soundbridge

    Public Sub New(ByVal soundbridge As Soundbridge)
        _soundbridge = soundbridge
    End Sub

    Public ReadOnly Property Soundbridge() As Soundbridge
        Get
            Return _soundbridge
        End Get
    End Property
End Class

File: src/Pixa.Soundbridge/SoundbridgeEventArgs.cs
using System;

namespace Pixa.Soundbridge
{
    public class SoundbridgeEventArgs : EventArgs
    {
        private Soundbridge _soundbridge;

        public SoundbridgeEventArgs(Soundbridge soundbridge)
        {
            _soundbridge = soundbridge;
        }

        public Soundbridge Soundbridge
        {
            get
            {
                return _soundbridge;
            }
        }
    }
}